readme.md

```diff
@@ -27,37 +27,37 @@
 ## Миграции
```

Comment:
Let's also translate the readme file. Just like we translate such files in the handbook.

* There should be two files: readme.md (in English) and readme-ru.md (in Russian).
* Add links `🇷🇺 Switch to Russian` and `🇬🇧 Switch to English`